apps/purduebasketball/purdue_basketball.star

```diff
@@ -0,0 +1,75 @@
+
+def main(config):
+    year = time.now().year
+    api_key = config.str("api_key", "")
+
+    cbb_stat_endpoint = "https://api.sportsdata.io/v3/cbb/scores/json/TeamSeasonStats/" + str(year) + "?key=" + api_key
+
+    if (api_key == ""):
+        return render.Root(
+            child = render.Text("API KEY Needed"),
+        )
+
+    purdue_stat = get_purdue_stat(cbb_stat_endpoint)
+    wins = int(purdue_stat["wins"])
+    losses = int(purdue_stat["losses"])
+    # child = render.Text("{}-{}".format(wins,losses))
+
+    return render.Root(
+        child = render.Box(
+            # width=48,
+            padding = 5,
+            child = render.Column(
+                children = [render.Row(
+                    children = [
+                        render.Image(src = purdue_logo, width = 24),
+                        render.Text("{}-{}".format(wins, losses)),
+                    ],
+                    main_align = "space_between",
+                    cross_align = "center",
+                    expanded = True,
+                )],
+                cross_align = "center",
+            ),
+        ),
+    )
+
+def get_purdue_stat(endpoint):
+    data = http.get(endpoint)
```

Comment:
You can add some caching here by passing the `ttl_seconds` parameter to `http.get`. Sportsdata recommends 5 minutes between calls, so `ttl_seconds=60*5` should be good.